#This represent three different representation of a trasform, 
#RPY, axis-angle and homogeneus trasformation matrix
string parent_frame
string child_frame
geometry_msgs/Vector3 translation
geometry_msgs/Vector3 rpy
AxisAngle axis_angle
TransformationMatrix tf_matrix